### PR TITLE
Add mode for when only 'from'-station is specified

### PR DESCRIPTION
**Pull request by blixhavn**

### CODE REVIEW

File: MMM-Bysykkel.js [hunk 1]
```diff
@@ -34,42 +34,68 @@ Module.register("MMM-Bysykkel", {
...
+  displaySingleStation: function(wrapper) {
+
+    // Bottom section with from, eta and to
+    const bottom = document.createElement("div");
+    bottom.className = "bottom";
+
+    const from = this.createInfoSection(
+      "bike",
+      this.fetchedData.from.available,
+      this.fetchedData.from.total
```
Comment (JonasTriki) [inline MMM-Bysykkel.js]: Perhaps add an option to include the name when displaying a single station, for consistency?

File: README.md [hunk 2]
```diff
@@ -53,7 +58,7 @@ These are the valid configuration options you can put inside the config array ab
...
 | city                 | What city we're biking in.                                                                                                                                                                                 | "bergen"                      |
 | fromStationId        | Desired starting station identifier; used to tell which station we're starting from.                                                                                                                       | 3                             |
-| toStationId          | Desired end station identifier; used to tell which station we're heading towards.                                                                                                                          | 5                             |
+| toStationId          | Desired end station identifier; used to tell which station we're heading towards. If set to 0, the module will display a simple variant (see screenshot 2).                                                                                                                          | 5                             |
```
Comment (JonasTriki) [inline README.md]: Can you change the `toStationId` to -1 for single stations (to ensure the ID does not conflict with future stations)?

File: cities/urbancity.js [hunk 3]
```diff
@@ -118,13 +121,16 @@ const UrbanCity = {
...
         self.data.from.available = fromStationStatus.num_bikes_available;
-        self.data.to.available = toStationStatus.num_docks_available;
+
+        if (config.toStationId) {
+          const toStationStatus = self.findStation(
+            statusJson,
+            config.toStationId
+          );
+          self.data.to.available = toStationStatus.num_docks_available;
+        }
```
Comment (JonasTriki) [inline cities/urbancity.js]: Same as before (-1 indicates single station); change to
```javascript
if(config.toStationId !== -1) {
...
}
```

File: cities/urbancity.js [hunk 4]
```diff
@@ -51,28 +51,31 @@ const UrbanCity = {
...
+          const toStationInfo = self.findStation(
+            infoJson,
+            config.toStationId
+          );
+          data.to = {
             name: toStationInfo.name,
             total: toStationInfo.capacity
           }
-        };
+        }
```
Comment (JonasTriki) [inline cities/urbancity.js]: Same as before (-1 indicates single station); change to
```javascript
if(config.toStationId !== -1) {
...
}
```

**Review by JonasTriki — CHANGES_REQUESTED** [review]
Thank you for the PR! Please read over my attached comments below.

File: MMM-Bysykkel.js (hunk 1 above)
Comment (blixhavn) [reply MMM-Bysykkel.js]: Added the option `displaySingleStationName`, with default value `false`. Ok to you?

**Comment by blixhavn** [comment]
I fixed the issues you pointed out, and also made a few other changes. For the `this.file("style.css")`, I simply couldnt get my local MagicMirror to work without it. Also, I wrapped all your CSS in the module class name to avoid collisions.

**Review by JonasTriki — APPROVED** [review]
LGTM!

**Pull request merged**